In [1]:
import numpy as np

## Задача 1

Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя уровень значимости 0.025 и правостороннюю критическую область, проверить данную гипотезу при условии, что в выборке из 100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 мм.

Распределение выборочного среднего достаточно похоже на нормальное распределение с параметрами $a = M(X)$ и $\sigma = \sqrt{\dfrac{D(X)}{n}}$.

При уровне значимости 0.025 и правосторонней критической области, $\alpha = 0.025*2 = 0.05$
Подходит правило двух сигм и правый "хвост". 

In [6]:
# Критическая область
m = 17
n = 100
𝜎_m = np.sqrt(4/n)
pr = m + 2 * np.sqrt(4/n)
pr

17.4

Правосторонняя критическая область получается в промежутке:

$( 17.4, + \infty)$

17.5 - попадает в эту область, значит гипотезу о среднем диаметре в 17мм при уровне значимости 0.025 считаем неверной

## Задача 2

Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочное среднее равно 80, а объём равен 256.

Доверительный интервал для надежности 0.95 формируем по правилу 2-х сигм

In [11]:
m = 80
n = 256
d = 16
(m - 2 * np.sqrt(d / n) , m + 2 * np.sqrt(d / n))

(79.5, 80.5)

## Задача 3

Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:

202, 203, 199, 197, 195, 201, 200, 204, 194, 190

Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

При доверительной вероятности 99% и n = 10 t = 3.24983554402 

In [14]:
samples = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])
m = samples.mean()
s = samples.std(ddof=1)
n = 10
t = 3.24983554402 
(m - t * s / np.sqrt(n), m + t * s / np.sqrt(n))

(193.923228400357, 203.076771599643)

При заданных значениях можно считать утверждение, что средний вес пачки печенья составляет 200 г., верным